css/portal-dashboard/all-responses-popup/popup-header.less

```diff
@@ -149,7 +150,7 @@
           background-color: @feedback-green-light2;
           border-color: @feedback-green;
           width: 204px;
-          left: calc(100vw/2);
+          left: calc(112vw/2);
```

Comment:
These calculations seem a little odd (using 112 vw for example), but they seem to work.  We might make a note to try to dig back into these to make sure they work at different screen sizes.

Reply:
Ah, I see what's happening here.  Might have been easier to position relative to the parent container instead of the viewport.  I wouldn't worry about it since it displays ok.  If I'm ambitious, I might do a quick refactor when I update the SVGs.

Reply:
My thought was to have them move depending on screen size. If I position them relative, they stay static to the left and right spec. Not sure what we want. I'll ask Michael.